cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -146,19 +146,19 @@ func (cfm *configFetchMeasurementsCmd) parseFetchMeasurementsFlags(cmd *cobra.Co
 	if err != nil {
 		return &fetchMeasurementsFlags{}, err
 	}
-	cfm.log.Debugf("Parsed measurements URL")
+	cfm.log.Debugf("Parsed measurements URL as %q", measurementsURL.String())
 
 	measurementsSignatureURL, err := cfm.parseURLFlag(cmd, "signature-url")
 	if err != nil {
 		return &fetchMeasurementsFlags{}, err
 	}
-	cfm.log.Debugf("Parsed measurements signature URL")
+	cfm.log.Debugf("Parsed measurements signature URL as %q", measurementsSignatureURL.String())
```

Comment:
measurementsURL can be nil, that's why the unit test fails. We also already log this in parseURLFlag(), so it think we can just delete these two lines.